test/factories/author.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :author do
+    sequence(:name) {|n| "Author #{n}" }
+    description { "#{name} description\n" }
+
+    site
+  end
+
+  factory :author_with_photo, parent: :author do
+    photo { Rack::Test::UploadedFile.new(Rails.root + "test/fixtures/images/face.png", "image/png") }
```

Comment:
Alternatively, the example of Machinist describes:
``` ruby
User.blueprint do
  avatar { File.open("#{Rails.root}/test/fixtures/files/rails.png") }
end
```

I think it is not important using `Rack::Test::UploadedFile` for FactoryGirl usage.
I like simple code.